seed/seed.js

```diff
@@ -0,0 +1,208 @@
+        price: 200000,
+        image: 'download.jpeg',
+      }),
+      Product.create({
+        model: 'Urus Pearl Capsule',
+        description:
+          'The Urus Pearl Capsule offers a selection of exclusive pearl paints and style elements that embrace the bright colors of Lamborghini tradition. This new realm of customization showcases the inimitable style and exhilarating performance of the world’s first Super Sport Utility Vehicle. Take the freedom of Urus to a higher level by forging new paths as an expression of yourself. Unlock any road, unlock your personality.',
+        price: 230000,
+        image: 'download.jpeg',
+      }),
+      Product.create({
+        model: 'Urus Graphite Capsule',
+        description:
+          'The Urus Graphite Capsule presents Lamborghini drivers all the possibilities to go anywhere with inimitable character and style. Sporting a powerful look and a dynamic stance, these exclusive configurations combine sophisticated satin-effect matte colors with contrasting bright-colored details that express new and alluring interpretations of your personality.It’s time to reveal yourself to the fullest. Venture into new challenges and declare your most authentic self aboard the world’s first Super SUV. Unlock any road … unlock your personality.',
+        price: 250000,
+        image: 'download.jpeg',
+      }),
+      Product.create({
+        model: 'Sian FKP 37',
+        description:
+          'The Sián FKP 37 is the first super sports car powered by a V12 engine and hybrid technology based on supercapacitors. Its powerful V12 engine, coupled with electric boost, creates an unrivaled gem of engineering and technology. Sián—lightning in Bolognese—is a name that captures the car’s true character, foremost its speed, which exceeds 220 mph (350 km/h).',
+        price: 3150000,
+        image: 'download.jpeg',
+      }),
+    ]);
+
+    const orders = await Promise.all([
+      Order.create({
+        payment_date: '12/04/2019',
+        order_date: '12/04/2019',
+        shipping_date: '12/10/2019',
+        shipping_method: 'UPS',
+        order_status: 'processing',
+      }),
+    ]);
+
+    const shipments = await Promise.all([
+      Shipment.create({
+        tracking_num: '11111',
+        shipment_date: '09/23/2020',
+      }),
+    ]);
+    const carts = await Promise.all([
+      Cart.create({
+        session_id: '12345',
+        created_date: '11/11/2020',
+      }),
+    ]);
+
+    const payments = await Promise.all([
+      Payment.create({
+        card_type: 'Visa',
+        card_number: 42423332222444,
+        card_exp_month: 11,
+        card_exp_year: 25,
+        card_security_code: 222,
+      }),
+    ]);
+  } catch (err) {
+    console.log(err);
+  }
+}
+module.exports = seed;
+if (require.main === module) {
```

Comment:
wow this is fun

Reply:
lol. I felt the same way when I saw it. lol